ldap_jwt_auth/auth/jwt_handler.py

```diff
@@ -54,8 +55,15 @@ def refresh_access_token(self, access_token: str, refresh_token: str):
         """
         logger.info("Refreshing access token")
         self.verify_token(refresh_token)
+
         try:
             payload = self._get_jwt_payload(access_token, {"verify_exp": False})
+
+            authentication = Authentication()
```

Comment:
I think the comment above could do with updating.

Reply:
Addressed in https://github.com/ral-facilities/ldap-jwt-auth/pull/23/commits/ddf3c92ef5eaeb252d3263c182861744dacfdf14